common/src/main/resources/example_config.json

```diff
@@ -45,7 +45,12 @@
       "Pochita",
       "Chainsaw man",
       "Risky endeavour",
-      "Safety first"
+      "Safety first",
+      "Precision Incision",
```

Comment:
I prefer the titles to only have the first capital letter, like "Precision incision"

Reply:
fair enough